src/components/Overview.js

```diff
@@ -1,11 +1,17 @@
 import React, { Component } from 'react'
+import DeleteTask from './DeleteTask'
 
 class Overview extends Component {
   render() {
-    const { tasks } = this.props
+    const { tasks, handleDelete } = this.props
 
     const list = tasks.map((task, i) => {
-      return <li key={i}>{i + 1 + '. ' + task.text}</li>
+      return (
+        <li key={task.id}>
+          <span>{i + 1 + '. ' + task.text}</span>
```

Comment:
Hier könnte man cooler mit einem String formatter arbeiten ;)

Reply:
Do you mean with string formatter e.g. template string? Like this: {`${i+1}. ${task.text}`}

Reply:
Yep :)

Reply:
Done through amend and force push.